main.js

```diff
@@ -64,6 +65,45 @@ function checkStatus() {
             setConnected(result.alive);
         }
     });
+
+    // Check other read only objects
+    // TODO: This should probably be in it's own function
+    var postData = JSON.stringify({
+		'method' : 'getPowerStatus',
+		'params' : [''],
+		'id' : 1,
+		'version' : '1.0'
+	});
+
+	var options = {
+		host: adapter.config.ip,
+		port: '80',
+		path: '/sony/system',
+		method: 'POST',
+		headers: {
+			'Content-Type': 'application/x-www-form-urlencoded',
+			'Content-Length': postData.length
+		}
+	};
+            
+	var postReq = http.request(options, function(postRes){
+		var body = '';
+		postRes.on('data', function(data){
+			body += data;
+		});
+		
+		postRes.on('end', function(){
+			var states = JSON.parse(body);
```

Comment:
You should put a try/catch block around JSON.parse to increase error tolerance.

Reply:
No problem.